src/css/components/panel.scss

```diff
@@ -29,30 +28,32 @@ $panel-padding: $grid-3;
 
 }
 
-// TODO this is a hack, panel-row borders need to be reconciled, panel-row vs
-// panel-row-bordered
-.panel-row-clean {
-  border: none !important; // stylelint-disable-line declaration-no-important
-}
-
-h1 + .panel,
-h2 + .panel,
-h3 + .panel,
-h4 + .panel {
-  margin-top: 2rem;
+.panel-header {
+  align-items: flex-end;
+  display: flex;
 }
 
 .panel-title {
+  flex: 1 0 0;
   font-size: $sans-s7;
   font-weight: 400;
   line-height: $lh-single;
-  margin-bottom: $grid-3;
+  margin-bottom: $grid-5;
 }
 
 .panel-secondary {
```

Comment:
What is panel secondary? What's it's purpose? I think defining that and naming it so it matches it's purpose might make it easier to use.